cl/settings/third_party/elasticsearch.py

```diff
@@ -53,9 +53,13 @@
     "ELASTICSEARCH_PASSWORD",
     default="password",
 )
+ELASTICSEARCH_VERIFY_CERT = env(
+    "ELASTICSEARCH_VERIFY_CERT",
+    default=False,
+)
 ELASTICSEARCH_CA_CERT = env(
     "ELASTICSEARCH_CA_CERT",
-    default="/opt/courtlistener/docker/elastic/ca.crt",
```

Comment:
I’m fairly confident that this is given a real value in production. However, in the unlikely event that this is not, `cl-django` will not be happy that it disappeared (but will tolerate this since certificates are not verified).

Silver lining is that it will reveal an issue that should be fixed? (the private key for `cl-es.crt` is in the public repo).

Reply:
Actually, I believe we don't have the `ELASTICSEARCH_CA_CERT` value set in production. @mlissner could help confirm this.

The reason for this is that we decided to skip certificate verification in production because we were unable to obtain the correct `CA` for verifying self-signed certificates, as described in https://github.com/freelawproject/courtlistener/issues/2958 so we just set `verify_certs` to `False`

I think we have two options moving forward:

- We could try again to obtain the correct `CA` from the ES cluster to use alongside the ALB's certificate, which would allow us to verify certificates in production.
- We could continue to skip certificate verification in production, as this is the default behavior set by `ELASTICSEARCH_VERIFY_CERT`.

Reply:
Yeah, pretty sure that's not set.